Source/CMakeLists.txt

```diff
@@ -36,6 +36,7 @@ endif ()
 
 if (ENABLE_WEBCORE)
     add_subdirectory(WebCore)
+    add_custom_target(WebCoreSerializationFiles DEPENDS ${WebCore_SERIALIZATION_IN_FILES_FULL_PATH})
```

Comment:
This line certainly shouldn't go here, and it probably isn't necessary.

Reply:
The goal was to have a target which causes the generation to happen again if the serialization.in files change in webcore.  The custom target is here because then the webkit subdirectory cmake file can see it.  If this isn't needed I'm happy to remove it.